include/windows_aligned_file_reader.h

```diff
@@ -30,18 +30,22 @@ class WindowsAlignedFileReader : public AlignedFileReader {
 
   // Open & close ops
   // Blocking calls
-  DISKANN_DLLEXPORT virtual void open(const std::string &fname);
-  DISKANN_DLLEXPORT virtual void close();
+  DISKANN_DLLEXPORT virtual void open(const std::string &fname) override;
+  DISKANN_DLLEXPORT virtual void close() override;
 
-  DISKANN_DLLEXPORT virtual void register_thread();
-  DISKANN_DLLEXPORT virtual void deregister_thread() {
+  DISKANN_DLLEXPORT virtual void register_thread() override;
+  DISKANN_DLLEXPORT virtual void deregister_thread() override {
+      // TODO: Needs implementation.
```

Comment:
There should be a windows_aligned_file_reader.cpp that has the implementations?

Reply:
It may be in some other branch, it wasn't here.